pkg/storagepool/cstorpool_template_engine.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
this file is not required. The caller code can make use of `pkg/engine/cas_template_engine.go` file